README.md

```diff
@@ -34,24 +168,17 @@ scrape_configs:
 
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
   - job_name: 'maia'
-    metrics_path: "/api/v1/metrics"
+    metrics_path: "/federate"
```

Comment:
True for Prometheus. But for Maia, I think it's part of the versioned API, as the same logic is applied as with the other methods. It's not the plain `/federate` endpoint. Using it unversioned would prevent (breaking) changes to the underlying logic. E.g. if the format of the basic-auth params needs to be changed, etc.
Makes sense?